frontends/web/src/components/layout/header.module.css

```diff
@@ -132,4 +131,20 @@
     .guideIcon {
         font-size: var(--size-small);
     }
+}
+
+@media (max-width: 560px) {
+    .container .header.iconCollapsed {
+        flex-wrap: wrap;
+        padding: var(--space-half);
+    }
+    
+    .title.iconCollapsed {
+        width: 100%;
+    }
+    
+    .children.iconCollapsed {
+        width: 100%;
+        margin-top: calc(var(--space-quarter) + var(--space-eight));
+    }
```

Comment:
I played around a little (just with the CSS part), I think with this changes it could just wrap automatically (always without prop)

```diff
diff --git a/frontends/web/src/components/layout/header.module.css b/frontends/web/src/components/layout/header.module.css
index 52f02ef9d..7f40b9050 100644
--- a/frontends/web/src/components/layout/header.module.css
+++ b/frontends/web/src/components/layout/header.module.css
@@ -2,7 +2,7 @@
     width: 100%;
 }
 
-.container.fixed {
+.container.fixed { /* TODO: unused remove? */
     position: fixed;
     left: var(--sidebar-width);
     height: var(--header-height);
@@ -11,16 +11,16 @@
 }
 
 .header {
-    padding: 0 var(--space-default);
     max-width: var(--content-width);
     height: var(--header-height);
     width: 100%;
     display: flex;
     flex-direction: row;
-    justify-content: space-between;
+    justify-content: start;
     align-items: center;
-    flex: none;
     margin: 0 auto;
+    flex-wrap: wrap;
+    padding: var(--space-default);
 }
 
 .children {
@@ -28,6 +28,7 @@
     gap: var(--space-half);
     flex-direction: row;
     align-items: center;
+    margin-bottom: var(--space-quarter);
 }
 
 .children > a {
@@ -55,7 +56,9 @@
 
 .title {
     white-space: nowrap;
-    flex: 1;
+    flex-grow: 1;
+    flex-shrink: 0;
+    margin-bottom: var(--space-quarter);
 }
 
 .title > * {
@@ -71,10 +74,13 @@
 .sidebarToggler {
     width: 32px;
     height: 32px;
-    margin-right: var(--spacing-default);
     display: flex;
     flex-direction: row;
     align-items: center;
+    flex-basis: 32px;
+    flex-grow: 0;
+    flex-shrink: 0;
+    margin-bottom: var(--space-quarter);
 }
 
 .sidebarToggler img {
@@ -115,14 +121,13 @@
 }
 
 @media (max-width: 768px) {
+    .header {
+        padding: var(--space-half);
+    }
     .sidebarToggler.hideSidebarToggler {
         display: none;
     }
 
-    .container .header {
-        padding: 0 var(--space-half);
-    }
-
     .container .sidebarToggler {
         margin-right: 0;
     }
@@ -132,19 +137,3 @@
         font-size: var(--size-small);
     }
 }
-
-@media (max-width: 560px) {
-    .container .header.iconCollapsed {
-        flex-wrap: wrap;
-        padding: var(--space-half);
-    }
-    
-    .title.iconCollapsed {
-        width: 100%;
-    }
-    
-    .children.iconCollapsed {
-        width: 100%;
-        margin-top: calc(var(--space-quarter) + var(--space-eight));
-    }
-}
```